planning/autoware_route_handler/src/route_handler.cpp

```diff
@@ -403,17 +422,25 @@ void RouteHandler::setLaneletsFromRouteMsg()
     primitive_size += route_section.primitives.size();
   }
   route_lanelets_.reserve(primitive_size);
+  std::vector<RouteRtreeNode> rtree_nodes;
+  rtree_nodes.reserve(primitive_size);
+  auto i = 0UL;
 
   for (const auto & route_section : route_ptr_->segments) {
     for (const auto & primitive : route_section.primitives) {
       const auto id = primitive.id;
       const auto & llt = lanelet_map_ptr_->laneletLayer.get(id);
       route_lanelets_.push_back(llt);
+      rtree_nodes.emplace_back(
+        boost::geometry::return_envelope<autoware_utils_geometry::Box2d>(
+          llt.polygon2d().basicPolygon()),
+        i++);
       if (id == route_section.preferred_primitive.id) {
         preferred_lanelets_.push_back(llt);
       }
     }
   }
+  route_lanelets_rtree_ = RouteRtree(rtree_nodes);
```

Comment:
I don't know the `RouteRtree` implementation, but please consider whether it's possible to move it.

```suggestion
  route_lanelets_rtree_ = RouteRtree(std::move(rtree_nodes));
```

Reply:
Clang-tidy says `std::move` is unnecessary here.
```
Passing result of std::move() as a const reference argument; no move will actually happen (fix available)clang-tidy[performance-move-const-arg](https://clang.llvm.org/extra/clang-tidy/checks/performance/move-const-arg.html)
```